Contact/Contact.cs

```diff
@@ -66,10 +66,12 @@ public bool IsEqual(Contact contact, Contact contact2)
         private static string formatPhoneNumber(string phone)

```

Comment:
As mentioned, this would be a good candidate for an extension method

Reply:
Given that I am using the same function to format three different variables, I am unsure how to make this an extension method without making multiple extension methods for each phone variable.

Reply:
I'm not sure what you mean.  An extension method is for a specific type, not a variable.  It isn't any different than your private method other than you are passing in an instance of the type you want to operate on.  The 2 approaches to sharing this code would be:
1.  PhoneHelper.RemoveFormatting(myPhoneNumber)
2. myPhoneNumber.RemoveFormatting()

I personally prefer the 2nd syntax which uses an extension method.